extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -87,28 +86,23 @@ public static function canShowModule() {
 	 * @return string - wiki vertical
 	 */
 	public function getWikiVertical() {
-		wfProfileIn( __METHOD__ );
-
 		$verticalId = WikiFactoryHub::getInstance()->getVerticalId( $this->wg->CityId );
 		if ( empty( self::$verticals[$verticalId] ) ) {
 			$verticalId = WikiFactoryHub::VERTICAL_ID_OTHER;
 		}
 
 		$name = self::$verticals[$verticalId];
 
-		wfProfileOut( __METHOD__ );
-
 		return $name;
 	}
 
 	/**
 	 * Get monetization units
+	 * @param Title $title
 	 * @param array $params
 	 * @return array|false $result
 	 */
-	public function getMonetizationUnits( $params ) {
-		wfProfileIn( __METHOD__ );
-
+	public function getMonetizationUnits( Title $title, $params ) {
 		$log = WikiaLogger::instance();
 		$loggingParams = [ 'method' => __METHOD__, 'params' => $params ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "method" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)